simplifies READMEs and links to AVS docs (#19)
aerospike/proximus-examples: prism-image-search/README.md

performs a query a vector embedding for the provided text is generated and
Proximus performs Approximate Nearest Neighbor(ANN) search to find relevant results .

> ## Pre-requisites

Reviewer: ```suggestion
## Prerequisites
```

You don't have to know Aerospike to get started, but you do need the following:

1. A Python 3.10 - 3.11 environment and familiarity with the Python programming language (see [Setup Python Virtual Environment](./prism-image-search/README.md#setup-python-virtual-environment)).

> 1. An Aerospike Vector Search host (Sandbox, or local)

Reviewer: ```suggestion
1. An Aerospike Vector Search host (sandbox or local).
```

> ## Configure Proximus Host

Reviewer: ```suggestion
## Configure AVS (Proximus) Host
```

> If you are connecting to a sandbox trial you'll need to set the following. 

Reviewer: ```suggestion
If you are connecting to a trial sandbox, you'll need to set the following:
```

```shell
export PROXIMUS_HOST=<SANDBOX_IP>
```

## Link your photos
To index your local photos, create a symlink to a location with photos directory.

```shell
ln -s ~/Pictures prism/static/images/data
```
## Start the application

```shell
 waitress-serve --host 127.0.0.1 --port 8080 --threads 32 prism:app
```

## Performing a quote search
Navigate to http://127.0.0.1:8080 and perform a search for images based on a description. 

## Install using docker compose

> If you have a license key, you can easily setup Aerospike, Proximus, and the prism-image-search

Reviewer: ```suggestion
If you have a license key, you can easily setup Aerospike, AVS, and the prism-image-search
```

> app using docker-compose. When using docker-compose you'll need to place your images in `container-volumes`

Reviewer: ```suggestion
app using docker-compose. When using docker-compose, you'll need to place your images in `container-volumes`
```

```shell
ln -s ~/Pictures container-volumes/prism/images/static/data
```

### 1. Build the prism image 
```
cd prism-image-search && \\
docker build -t prism . -f Dockerfile-prism
```

### 2. Add features.conf
Proximus needs an Aerospike features.conf file with the vector-search feature enabled.
Add your features.conf file to container-volumes/proximus/etc/aerospike-proximus.

### 3. Start the environment
```
docker compose up
```
## Developing

> This demo is build using [Python Flask](https://flask.palletsprojects.com/en/2.3.x/)

Reviewer: ```suggestion
This demo is built using [Python Flask](https://flask.palletsprojects.com/en/2.3.x/)
```

> and [Vue.js](https://vuejs.org/). In order to developer follow the steps to 

Reviewer: ```suggestion
and [Vue.js](https://vuejs.org/). To start developing, follow the steps to 
```

> setup your Python environment.

Reviewer: ```suggestion
set up your Python environment.
```

> ### Setup Python Virtual Environment

Reviewer: ```suggestion
### Set up Python Virtual Environment
```
